defaults/main.yml

```diff
@@ -105,6 +105,7 @@ nextcloud_tls_session_cache_size: 50m  # in Byte or human readable size notation
 
 # [APPS]
 nextcloud_apps: {}
+nextcloud_disable_apps: {}
```

Comment:
When adding a new parameter, especially when default is empty, is very good if we have an actual example with a list in the [Example Playbooks](https://github.com/aalaesar/install_nextcloud#example-playbook).